app/models/duffel.rb

```diff
@@ -18,20 +18,22 @@ def count_of(item_id)
     contents[item_id.to_s]
   end
 
-  def subtotal
-    item_prices = contents.map do |item_id, quantity|
-      item = Item.find(item_id)
-      item.price * quantity
+  def duffel_items
+    contents.map do |item_id, quantity|
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.